### PR TITLE
feat(metrics): init route to return raw statuscake data

### PR DESCRIPTION
**Pull request by VirgileLyon**
In this PR 🚚  :

- I create a ./metrics route, it gives you access to the statuscakedata.
- I add tests (😨).
- I also add Travis CI.
- README.MD is update.

### CODE REVIEW

**Comment by clement-berard** [comment]
there is no empty line on some files :)

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Get access to this following information:
 
 ## Configure
 
-To confiugure the project, copy `config_example.json` on your `config.json` to add:
+To configure the project, we use npm `config` package, follow `config/default.json` example to define yours with:
```
Comment (clement-berard) [inline README.md]: you can add a link to https://www.npmjs.com/package/config ;)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "10"
+  - "12"
+cache:
+  yarn: true
+script:
+  - yarn test
```
Comment (clement-berard) [inline .travis.yml]: add the lint task here ;)

Comment (VirgileLyon) [reply .travis.yml]: For the next PR 🍡

Comment (clement-berard) [reply .travis.yml]: ok, you can do a 'npm run lint' to test this PR ;)

File: README.md (hunk 1 above)
Comment (VirgileLyon) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Get access to this following information:
 
 ## Configure
 
-To confiugure the project, copy `config_example.json` on your `config.json` to add:
+To configure the project, we use npm `config` package, follow `config/default.json` [https://www.npmjs.com/package/config](https://www.npmjs.com/package/config) example to define yours with:
```
Comment (clement-berard) [inline README.md]:
```suggestion
To configure the project, we use npm  [`config`](https://www.npmjs.com/package/config) package, follow `config/default.json` example to define yours with:
```

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (fdubost) [inline .travis.yml]: This line should be empty and contain no space 😇

File: README.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ Get access to this following information:
...
 - interval (Speedtime is check every second by default)
 - apikey (Imperative to get access to statuscake)
 - username (Imperative to get access to statuscake)
```
Comment (fdubost) [inline README.md]: The definition of `baseUrl`, `route` and `listener` is missing 👯

File: src/statuscake.js [hunk 6]
```diff
@@ -20,11 +22,13 @@ const fetchStatuCakeData = () => {
...
+        debug(URL, Title, loadTimeMS, fileSizeKB, Requests); 
+        Storage.setData(urls)
       });
     }).catch((error) => {
       logError('axios can not access to the URL', error);
     });
 };
 
 module.exports = fetchStatuCakeData;
+
```
Comment (fdubost) [inline src/statuscake.js]: This line should be empty 😈

File: src/___tests__/server.test.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+const request =require("supertest");
```
Comment (fdubost) [inline src/___tests__/server.test.js]: Seems to be some lint issues 😄

File: README.md (hunk 3 above)
Comment (VirgileLyon) [reply README.md]: Thanks 🖌

File: .travis.yml (hunk 4 above)
Comment (VirgileLyon) [reply .travis.yml]: Done, thanks 👍

File: src/statuscake.js (hunk 6 above)
Comment (VirgileLyon) [reply src/statuscake.js]: Done! 👼

File: src/___tests__/server.test.js (hunk 7 above)
Comment (VirgileLyon) [reply src/___tests__/server.test.js]: 😑

**Comment by VirgileLyon** [comment]
test this please

**Pull request merged**